sections/main-password-footer.liquid

```diff
@@ -75,13 +75,11 @@
     {%- endif -%}
   </ul>
   {% capture shopify %}
-    <a class="shopify-link" href="//www.shopify.com" rel="nofollow" target="_blank" title="Create your own online store with Shopify">
+    <a class="shopify-link" href="//www.shopify.com" rel="nofollow" target="_blank" aria-describedby="warning-text" aria-label="Shopify">
```

Comment:
My bad for not catching it sooner but since we're using a different ID now it needs to be updated here as well:
```suggestion
    <a class="shopify-link" href="//www.shopify.com" rel="nofollow" target="_blank" aria-describedby="a11y-new-window-message" aria-label="Shopify">
```